### DISCUSSION TITLE
Feature: Have a drop down option in the Cheat Menu to choose which entertainers you want in your park

### DISCUSSION THREAD

**Discussion by tcek** [OpenRCT2/OpenRCT2#15753] [Feature requests]
**OS:** [e.g. Windows 10]
**Version:** [e.g. 0.3.4]
**Commit/Build:** [e.g. 426e106]

I'm not sure if this is the right place to put it, but I feel it would make things so much simpler and easier to click on the entertainers you want in your park through the Cheats menu or even in the Options menu, instead of manually having to type it out in the console command which takes time. I think it's a good idea that will save many players some time when creating a park. 

Thanks in advance.

**Comment by jensj12** [comment]
You can already change the type of entertainers quite easily by clicking the entertainer and using the dropdown provided there. Is that what you meant?

**Comment by tcek** [comment]
I provided a screenshot for you. It doesn't show all of the entertainers. That is where you have to go in to console command and manually type their ID's in and a bunch of other stuff.

![entertainer](https://user-images.githubusercontent.com/21962525/126515501-a119992b-e003-47cf-ad9d-b1af1f1fecfc.jpg)

**Comment by duncanspumpkin** [comment]
I think what you are asking for is to be able to override the scenario selected entertainers? That should be doable with some minor changes.
Would require overriding `staff_get_available_entertainer_costumes`

**Comment by tcek** [comment]
That's it. I think they will be a great feature :) thanks.

**Comment by Gymnasiast** [comment]
Could be tied to "ignore research status" or some similar cheat, I think.
